examples/gallery/embellishments/rose.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
According to this comment: https://github.com/GenericMappingTools/pygmt/issues/876#issuecomment-791649980, the `rose` example should be put into the "Histogram" category.

Reply:
Ok fine, to keep track will open a new issue regarding that point.